extensions/wikia/ImageReview/ImageStateUpdater.class.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+class ImageStateUpdater extends WikiaModel {
```

Comment:
Another more defined class where all the code was pulled directly from the `ImageReviewHelper` class.